### ISSUE TITLE
[BUG] longintrepr.h not found

### ISSUE THREAD

**Issue by ericsnowcurrently** [cython/cython#4461]
**Describe the bug**

An extension using Cython with CPython main won't build:

```
gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -O0 -fPIC -I/home/esnow/projects/work/cpython-perf/uvloop/uvloop-dev-3.11/include -I/home/esnow/projects/work/cpython-perf/cpython/Include -I/home/esnow/projects/work/cpython-perf/cpython -c aiohttp/_websocket.c -o build/temp.linux-x86_64-3.11/aiohttp/_websocket.o
  aiohttp/_websocket.c:198:12: fatal error: longintrepr.h: No such file or directory
     #include "longintrepr.h"
              ^~~~~~~~~~~~~~~
```

**To Reproduce**

Try to build https://github.com/MagicStack/uvloop.

**Environment (please complete the following information):**
 - OS: Linux
 - Python version 3.11.0 (pre-alpha)
 - Cython version 0.29.24

**Additional context**

About a month ago, Include/longintrepr.h was moved to Include/cpython/longintrepr.h.  See https://github.com/python/cpython/pull/28968 (https://bugs.python.org/issue35134).

**Comment by ericsnowcurrently** [comment]
Looks like this was already fixed in cython/cython#4428.

**Comment by colinfinn** [comment]
a
